warehouse-management-service/internal/handler/router.go

```diff
@@ -0,0 +1,17 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/middleware"
+	"github.com/go-chi/chi/v5"
+)
+
+func (h *handler) router() http.Handler {
```

Comment:
question (non-blocking): Is there a reason we are keeping the private methods in a separate file?

Reply:
This could have well resided in handler.go
I've chosen to place it in a separate file,
1. for the convenience of locating all the routes 
2. the dependency of the router library is isolated to this file